### PR TITLE
[onert/ggml] Don't specify arch on armv7l linux build

### PR DESCRIPTION
**Pull request by glistening**
It removes the specific arch on ggml cross armv7l linux build. Previously, ggml specifies -mfpu=neon-fp-armv8, which may cause illegal instruction signal during running ggml_quantize_row_q8_0, especially compiled with gcc-13 -On, where n = 1,2,3 on odroid xu4. (ARMv7 Cortex-A15).

ONE-DCO-1.0-Signed-off-by: Sanggyu Lee <sg5.lee@samsung.com>

Related: #14391

### CODE REVIEW

File: runtime/3rdparty/ggml/src/CMakeLists.txt [hunk 1]
```diff
@@ -1098,7 +1098,7 @@ if (CMAKE_OSX_ARCHITECTURES      STREQUAL "arm64" OR
                 list(APPEND ARCH_FLAGS -mfpu=neon-vfpv4 -mno-unaligned-access -funsafe-math-optimizations)
             else()
                 # Raspberry Pi 2
```
Comment (glistening) [inline runtime/3rdparty/ggml/src/CMakeLists.txt:1100]: Our target is not Raspberry Pi 2.

**Review by chunseoklee — DISMISSED** [review]

File: runtime/3rdparty/ggml/src/CMakeLists.txt (hunk 1 above)
Comment (hseok-oh) [reply runtime/3rdparty/ggml/src/CMakeLists.txt:1100]: Please add comment about custom fix.

Ex)
https://github.com/Samsung/ONE/blob/ab777f05be9660026356cdb22519de9839409afd/runtime/3rdparty/ggml/CMakeLists.txt#L232

**Review by hseok-oh — APPROVED** [review]
LGTM

**Comment by glistening** [comment]
With this patch, on Ubuntu 24.04 mate:

```
$ Product/armv7l-linux.release/out/bin/onert_run -r 1 5.circle
Model Filename 5.circle
===================================
MODEL_LOAD   takes 19.119 ms
PREPARE      takes 442.078 ms
EXECUTE      takes 8.801 ms
- MEAN     :  8.801 ms
- MAX      :  8.801 ms
- MIN      :  8.801 ms
- GEOMEAN  :  8.801 ms
===================================
```

**Comment by hseok-oh** [comment]
I also confirmed this PR working on my environment.

**Comment by glistening** [comment]
@chunseoklee PTAL. Comment is updated as @hseok-oh requested.

**Review by chunseoklee — APPROVED** [review]

**Pull request merged**